pyquil/operator_estimation.py

```diff
@@ -852,74 +858,66 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
                       DeprecationWarning)
         symmetrize_readout = readout_symmetrize
 
+    symm_type = 0
+    if symmetrize_readout is not None:
+        if symmetrize_readout == 'exhaustive':
```

Comment:
do we think that having only one of the 5 choices also correspond to a particular string is necessary? maybe we could deprecate this for simplicity? how about making an [`IntEnum`](https://docs.python.org/3/library/enum.html#intenum) containing the various symmetrization options?

Reply:
I'm fine with deprecating. I would think we would want to match whatever `qc.run_symmetrized_readout` takes in as argument. If you think `IntEnum` is more appropriate, perhaps we should change it there as well?

Reply:
People are going to still pass an `int` to these functions, using the `IntEnum` would just be to aid readability to the branching sections of the source code -- they can be compared to `int`s, so they're basically just `int`s with a helpful label.